mibios/glamr/forms.py

```diff
@@ -2,7 +2,8 @@
 from django.core.exceptions import ValidationError
 from django.forms.widgets import Select, TextInput
 # from django.utils.safestring import mark_safe
-
+from django.forms import ModelForm
+from django.db.models import Q
```

Comment:
empty lines around django import block, within block import order is alphabetical
```suggestion
from django.db.models import Q
from django.forms import ModelForm

```